packages/whats-new/types.d.ts

```diff
@@ -0,0 +1,7 @@
+declare module '@automattic/whats-new' {
+	type FunctionProps = {
+		onClose: () => void;
+	};
+	function WhatsNewGuide( props: FunctionProps ): JSX.Element;
+	export default WhatsNewGuide;
+}
```

Comment:
Declared new type for this package, because now we're calling it from typescript